README.md

```diff
@@ -73,10 +74,4 @@ Add a prettier config in `package.json`:
   "trailingComma": "es5",
   "bracketSpacing": false
 }
-```
-
-Run `prettier-stylelint`:
-
-```
-prettier-stylelint **/*.scss
-```
+```
```

Comment:
A note about applying prettier fixes via `stylelint foo.css --fix` might be handy.